developers.libra.org/.gitignore

```diff
@@ -0,0 +1,21 @@
+
+# Distribution / packaging
+
+# OSX
+*.DS_Store
+
+# Installer logs
+
+# Unit test / coverage reports
+
+# Docusaurus
+website/build/
+website/i18n/
+website/node_modules/
+
+# libra/libra is cloned from github and used to extract crate READMEs and Coding Guidelibnes
+libra
+docs/crates/*
+docs/community/coding-guidelines.md
+docs/community/contributing.md
+website/static/docs/rustdocs/
```

Comment:
Probably want to double check that this is accurate given the changes you've made

Reply:
@bmwill Are you seeing something inaccurate? The should have just carried over from the `website` repo since they are relative paths. And will apply in the new `developers.libra.org` directory.

Reply:
I'm just saying you probably don't need to ignore 'libra' anymore and update the comment